kong/plugins/prometheus/exporter.lua

```diff
@@ -71,6 +75,23 @@ local function init()
                                          "Total bandwidth in bytes " ..
                                          "consumed per service/route in Kong",
                                          {"service", "route", "type"})
+
+
+  -- Hybrid mode status
+  if kong.version_num >= 2000000 then -- 200.00.00 -> 2.0.0 then
+    if kong.configuration.role == "control_plane" then
+      cp_metrics = true
+
+      clustering = require("kong.clustering")
+
+      metrics.dataplane_last_seen = prometheus:gauge("dataplane_last_seen",
+                                                "Last time data plane contacted control plane",
+                                                {"node_id", "hostname", "ip"})
+      metrics.dataplane_config_hash = prometheus:gauge("dataplane_config_hash",
+                                                "Config hash numeric value of the data plane",
+                                                {"node_id", "hostname", "ip"})
```

Comment:
Would adding the current expected config has as another metric series help here?
With that additional timeseries, one can compare which data planes are out of sync and which one are in sync.
cc @wyndigo

Reply:
Rethinking this, this seems wrong.
Control plane node should export a series on the expected config hash and data plane nodes should export the current config hash. That minimizes the moving parts and reflects the state of the world much more accurately. (In the current code, we are relying on the control-data plane communication to not be buggy and report the right config hash)

Reply:
@hbagdi That's good point. We can also export the current hash here.
I was thinking future metrics we can expose on DP side as well. The problem is if we want to compare metrics from different side, both metrics should contain same set of labels. Or we can `ignore()` labels but that seems putting more logic into prometheus. So i think though it's a good idea to expose the hash on DP side, making it available on CP is also worthwhile.

Reply:
We need to record of current config hash in control plane, there's currently no such logic in `clustering.lua`.
I'm adding a metrics for the DP for now, that will also benefit generic dbless kong.